### ISSUE TITLE
How pretrained model is training?

### ISSUE THREAD

**Issue by youngstu** [ubisoft/ubisoft-laforge-ZeroEGGS#31]
How pretrained model is training? 

I use data_pipeline_conf_v1.json and configs_v1.json to retrain model from scratch, but the result is not as good as official pretrained model.

Thanks.

**Comment by youngstu** [comment]
Could you give the training loss log for reference?
